platforms/ide/problems-api/src/main/java/org/gradle/api/problems/ProblemCloneBuilder.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.problems;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.problems.locations.ProblemLocation;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This is a copy of the ProblemCloneBuilder interface from platforms/ide/problems-api/src/main/java/org/gradle/api/problems/ProblemCloneBuilder.java
+ *
+ * @since 8.6
+ */
+@Incubating
+public interface ProblemCloneBuilder extends BuildableProblemBuilder,
+    ProblemBuilderDefiningDocumentation,
+    ProblemBuilderDefiningLocation,
+    ProblemBuilderDefiningLabel,
+    ProblemBuilderDefiningCategory {
+    ProblemCategory getCategory();
+
+    String getLabel();
```

Comment:
❓ 
Why do you need the getters here?

Reply:
I thought we should be able do consider the previous values when adding new ones.
But thinking about it, there is always the original problem, we can as for those :)